config/src/main/java/com/typesafe/config/ConfigValueFactory.java

```diff
@@ -60,10 +65,12 @@ private ConfigValueFactory() {
      *            object to convert to ConfigValue
      * @param originDescription
      *            name of origin file or brief description of what the value is
+     * @param comments
+     *            comments on this ConfigValue used when it's rendered
      * @return a new value
      */
-    public static ConfigValue fromAnyRef(Object object, String originDescription) {
-        return ConfigImpl.fromAnyRef(object, originDescription);
+    public static ConfigValue fromAnyRef(Object object, String originDescription, String[] comments) {
```

Comment:
hmm. well, another thought: why not make an overload that simply takes a ConfigOrigin, and then let's add public API to create a ConfigOrigin. Otherwise we will end up having to add tons more overloads potentially (for line number, filename, and any other fields we ever add to ConfigOrigin). If we add an overload that simply takes ConfigOrigin, it will automatically support everything ConfigOrigin does.